internal/services/datafactory/data_factory_pipeline_resource.go

```diff
@@ -109,6 +110,67 @@ func resourceDataFactoryPipeline() *pluginsdk.Resource {
 			},
 		},
 	}
+
+	if features.FourPointOhBeta() {
+		resource.Schema["parameters"] = &pluginsdk.Schema{
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"name": {
```

Comment:
Add a validation func here, the `name` should not be empty.

Reply:
added